google/datalab/contrib/pipeline/commands/_pipeline.py

```diff
@@ -27,12 +27,9 @@
 
 def _create_cell(args, cell_body):
```

Comment:
My understanding was there was the pipeline infrastructure/object model, but no %pipeline code.